src/util/handlers/ParallelHandler.ts

```diff
@@ -12,12 +12,10 @@ export class ParallelHandler<TIn = void, TOut = void> extends AsyncHandler<TIn,
   }
 
   public async canHandle(input: TIn): Promise<void> {
-    // eslint-disable-next-line @typescript-eslint/promise-function-async
```

Comment:
Huh, didn't know we had that 😂 . But yea, I would just add the async keyword to the functions below.